oak/server/rust/oak_runtime/src/node/wasm/mod.rs

```diff
@@ -798,21 +801,29 @@ impl WasmNode {
         node_name: &str,
         application_configuration: &ApplicationConfiguration,
         node_configuration: WebAssemblyConfiguration,
+        signature_configuration: &HashMap<String, SignatureConfiguration>,
     ) -> Result<Self, ConfigurationError> {
         let wasm_module_bytes = application_configuration
             .wasm_modules
             .get(&node_configuration.wasm_module_name)
             .ok_or(ConfigurationError::IncorrectWebAssemblyModuleName)?;
+
         let module = wasmi::Module::from_buffer(&wasm_module_bytes)
             .map_err(ConfigurationError::WasmiModuleInializationError)?;
         let entrypoint_name = node_configuration.wasm_entrypoint_name;
         validate_entrypoint(&module, &entrypoint_name).map_err(|err| {
             warn!("could not validate entrypoint: {:?}", err);
             ConfigurationError::IncorrectWebAssemblyModuleName
         })?;
+
+        let public_key = signature_configuration
+            .get(&node_configuration.wasm_module_name)
+            .map(|signature_item| signature_item.public_key.to_bytes());
```

Comment:
Each module may have more than one signature, right? So I think this should return a Vec, which would also make sense to replace the Option with a Vec in the `wasm_node_privilege` method.

Reply:
Done